libtenzir/test/loader_plugin.cpp

```diff
@@ -172,20 +172,6 @@ TEST(stdin loader - one complete chunk) {
                      str_chunk->begin(), str_chunk->end()));
 }
 
-TEST(file loader - parser deduction) {
-  CHECK_EQUAL(make_loader("file", "--timeout 1s foo.csv")->default_parser(),
-              "csv");
-  CHECK_EQUAL(make_loader("file", "--timeout 1s foo.ndjson")->default_parser(),
-              "json");
-  CHECK_EQUAL(make_loader("file", "--timeout 1s eve.json")->default_parser(),
-              "suricata");
-  CHECK_EQUAL(make_loader("file", "--timeout 1s -")->default_parser(), "json");
-  CHECK_EQUAL(make_loader("file", "--timeout 1s foo.txt")->default_parser(),
-              "json");
-  CHECK_EQUAL(make_loader("file", "--timeout 1s foo.")->default_parser(),
-              "json");
-}
-
```

Comment:
This is still supposed to work, no? Maybe it does again after your latest change?

Reply:
Oh, right, it should.